node/hubController.js

```diff
@@ -335,14 +334,14 @@ class HubController {
         }
 
         let customError = new OpenT2TError(err.statusCode, customMessage, err);
-        console.log(" custom error message:           " + customError.message);
-        console.log(" custom error name:           " + customError.name);
-        console.log(" custom error statusCode:           " + customError.statusCode);
-        console.log(" custom error innerError message:           " +  customError.innerError.message);
-        console.log(" custom error innerError stack:           " +  customError.innerError.stack);
+        this.ConsoleLogger.verbose(" custom error message:           " + customError.message);
+        this.ConsoleLogger.verbose(" custom error name:           " + customError.name);
+        this.ConsoleLogger.verbose(" custom error statusCode:           " + customError.statusCode);
+        this.ConsoleLogger.verbose(" custom error innerError message:           " + customError.innerError.message);
+        this.ConsoleLogger.verbose(" custom error innerError stack:           " + customError.innerError.stack);
```

Comment:
missed. Fixed  now.

---
In reply to: [103828244](https://github.com/openT2T/cloud/pull/26#discussion_r103828244) [](ancestors = 103828244)